linker-interface/shared/src/main/scala/org/scalajs/linker/interface/StandardConfig.scala

```diff
@@ -158,6 +158,22 @@ final class StandardConfig private (
 object StandardConfig {
   import StandardConfigPlatformExtensions.ConfigExt
 
+  def fingerprint(config: StandardConfig): String =
```

Comment:
This method should have `{}` to surround its body, since it spans more than 1 line.

Reply:
Done in 87b8de2b19bdc8b9e1518365a1a14733e87b6299